scripts/storage

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash
+
+##--------------------------------------------------------------------
+## Copyright (c) 2017 OSIsoft, LLC
+##
+## Licensed under the Apache License, Version 2.0 (the "License");
+## you may not use this file except in compliance with the License.
+## You may obtain a copy of the License at
+##
+##     http://www.apache.org/licenses/LICENSE-2.0
+##
+## Unless required by applicable law or agreed to in writing, software
+## distributed under the License is distributed on an "AS IS" BASIS,
+## WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+## See the License for the specific language governing permissions and
+## limitations under the License.
+##--------------------------------------------------------------------
+
+#
+# This script is used to call the PosegreSQL storage plugin script
+# to store and retrieve the sensor data when the database
+# is embedded in FogLAMP
+#
+
+#set -x
+
+
+PLUGIN_TO_USE=""
+
+
+# Logger wrapper
+storage_log() {
+    write_log "foglamp.script.storage" "$1" "$2" "$3" "$4"
+}
+
+
+## Get the default storage database plugin from the foglamp config file
+get_default_storage_plugin() {
+
+    # Remove new lines and store in a variable
+    json_foglamp=`tr -d '\n' < $1`
+
+    # Remove tabs
+    json_foglamp=`echo $json_foglamp | tr -d '\t'`
+
+    echo `echo $json_foglamp | grep -o '"default storage plugin" *:.*' | grep -o ':.*' | grep -o '".*' | cut -d'"' -f2`
```

Comment:
Using a Python script to extract this would be much easier and less error prone.